frontend/src/components/Util/TaskEditors/FloatingTaskEditor.module.scss

```diff
@@ -5,7 +5,6 @@
   box-shadow: 0 5px 12px 0 rgba(0, 0, 0, 0.3);
   transition: width 0.1s linear;
   max-height: 400px;
-  overflow-y: auto;
```

Comment:
This is not good. It will cause ugly overflows:
<img width="576" alt="Screen Shot 2020-10-09 at 19 58 47" src="https://user-images.githubusercontent.com/4290500/95640260-eaad4580-0a69-11eb-95fe-606c84286d9a.png">